internal/ident/runtime.go

```diff
@@ -0,0 +1,12 @@
+package ident
+
+import (
+	"google.golang.org/protobuf/compiler/protogen"
+)
+
+// Runtime idents.
+//
+//nolint:gochecknoglobals
+var (
+	RuntimeGOOS = protogen.GoIdent{GoName: "GOOS", GoImportPath: "runtime"}
```

Comment:
Not sure what the convention is in this package, but I would consider simply inlining this

Reply:
or at least make it a `func` to avoid declaring a global

Reply:
I haven't used protogen before and haven't really dug into exactly why e.g. all the `time.Second`, `time.Now` (and friends) have been wrapped in `protogen.GoIdent`. But could it be that you cannot inline something if you need to access another package?

Do you think it's fine to inline this?
I can try that, of course.

Reply:
The call to `QualifiedGoIdent` on the `protogen.GeneratedFile` will automatically track and insert an import statement at the right place, so you don't have to code-generate the imports yourself! It's quite nifty, all you need to do is to tell the generator the package name and the identifier to import - that's what we're doing here.

I usually do this inline in the code. Here is a snippet from what our team is working on right now:

```go
	g := gen.plugin.NewGeneratedFile(path, file.GoImportPath)
	g.P("// Package ", file.GoPackageName, " contains generated HTML templates.")
	g.P("//")
	g.P("// The HTML templates are generated from the schemas in the following protobuf package:")
	g.P("// ", file.Desc.Package(), ".")
	g.P("package ", file.GoPackageName)
	g.P()
	g.P("// Code generated by protoc-gen-go-html-template. DO NOT EDIT.")
	g.P()
	g.P("//go:embed *.tmpl")
	g.P("var templates ", g.QualifiedGoIdent(protogen.GoIdent{GoImportPath: "embed", GoName: "FS"}))
	g.P()
	g.P("// TemplateFiles returns the source files for the generated protobuf templates in this package.")
	g.P("func TemplateFiles() ", g.QualifiedGoIdent(protogen.GoIdent{GoImportPath: "io/fs", GoName: "FS"}), "{")
	g.P("return templates")
	g.P("}")
	g.P()
	g.P("// Templates returns the generated protobuf templates in this package.")
	g.P("func Templates() (*", protogen.GoIdent{GoImportPath: "html/template", GoName: "Template"}, ", error) {")
	g.P(`return template.ParseFS(templates, "*.tmpl")`)
	g.P("}")
```

As you can see, we don't need to worry about the imports, the code generation utility will instead that for us, if we use the `QualifiedGoIdent` method call correctly.

Reply:
Ok, have another look please! 😄